src/code/z_kanfont.c

```diff
@@ -39,8 +39,7 @@ void Font_LoadOrderedFont(Font* font) {
             loadOffset = 0;
         }
 
-        DmaMgr_SendRequest0(writeLocation, (uintptr_t)SEGMENT_ROM_START(nes_font_static) + loadOffset,
-                            FONT_CHAR_TEX_SIZE);
+        DmaMgr_SendRequest0(writeLocation, SEGMENT_ROM_START(nes_font_static) + loadOffset, FONT_CHAR_TEX_SIZE);
```

Comment:
So does SEGMENT_ROM_START_OFFSET work here or not? Either way I think it confuses the operation here.

Reply:
Weirdly no, this place requires the cast to happen before the arithmetic and not before. The devs were definitely not consistent with this